kart/add_dataset.py

```diff
@@ -69,6 +73,14 @@ def add_dataset(ctx, table_name, message, launch_editor, output_format):
         table_name, meta_items
     )
 
+    # Create a table object from the table_name:
+    path = TableV3.dataset_path_to_table_name(table_name)
```

Comment:
I'm not sure what this line adds - table_name is already a table name so it seems like this line converts it the wrong way and basically isn't required?